src/TripSegment/README.md

```diff
@@ -0,0 +1,64 @@
+# Trip Segment
+
+To implement Trip Segment component into your project you'll need import component itself and pass any other component you want as children.
+(#trips):
+
+```jsx
+import TripSegment from "@kiwicom/orbit-components/lib/TripSegment";
+import List from "@kiwicom/orbit-components/lib/List";
+import ListItem from "@kiwicom/orbit-components/lib/List/ListItem";
+```
+
+After adding import into your project you can use it simply like:
+
+```jsx
+<TripSegment>
+  <List>
+    <ListItem>Hello, World!</ListItem>
+    <ListItem>Lorem Ipsum</ListItem>
+  </List>
+</TripSegment>
+```
+
+## Props
+
+Table below contains all types of the props available in the Modal component.
+
+| Name     | Type                               | Default     | Description                                               |
+| :------- | :--------------------------------- | :------     | :-------------------------------------------------------- |
+| children  | `React.Node`                      |             | The content of the toggled component.
+| carrier   | [`Carrier[]`](#carrier)           |             |
+| duration  | `string` or `number`              |             | Flight duration.
+| departure | [`Airport{}`](#airport)           |             | Departure data of a flight.
```

Comment:
Plz make two props for departure: `departure: string` which will contain city name and city code and `departureTime: string` for time. Same case for `arrival`